src/ring_theory/valuation/valuation_subring.lean

```diff
@@ -715,6 +715,43 @@ set.subset_set_smul_iff
 
 end pointwise_actions
 
+section
+
+variables {L : Type*} [field L] [algebra K L] (K)
+
+open_locale pointwise
+
+/-- The decomposition subgroup defined as the stabilizer of the action
+on the type of all valuation subrings of the field. -/
+def decomposition_subgroup (A : valuation_subring L) :
```

Comment:
```suggestion
@[reducible] def decomposition_subgroup (A : valuation_subring L) :
```
to make it less annoying to swap between `decomposition_subgroup` and `stabilizer`.